Utilities/build-script-helper.py

```diff
@@ -9,6 +9,29 @@
 import shutil
 import subprocess
 import sys
+import json
+
+def error(message):
+  print("--- %s: error: %s" % (os.path.basename(sys.argv[0]), message))
```

Comment:
Should this go to stderr?

Reply:
Good idea. SwiftPM (where I copied the function from) prints to stdout but stderr seems like the better choice.